<ui version="4.0" >
 <class>helpViewerForm</class>
 <widget class="QWidget" name="helpViewerForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="mainTab" >
     <property name="tabPosition" >
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="indexTab" >
      <attribute name="title" >
       <string>&amp;Index</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>&amp;Look for:</string>
         </property>
         <property name="buddy" >
          <cstring>indexEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="indexEdit" />
       </item>
       <item>
        <widget class="QListView" name="indexListView" />
       </item>
       <item>
        <widget class="QLabel" name="helpSuggestions" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string>Help suggestions (based on the editor):
&lt;ul>
&lt;li>&lt;a href="qstring.html" title="The QString class provides a Unicode character string">QString&lt;/a>&lt;/li>

&lt;li>&lt;a href="qwidget.html">QWidget&lt;/a>&lt;/li>

&lt;li>&lt;a href="qapplication.html">QApplication&lt;/a>&lt;/li>

&lt;li>&lt;a href="qobject.html">QObject&lt;/a>&lt;/li>

&lt;ul>
&lt;br></string>
         </property>
         <property name="textFormat" >
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="popularPages" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="textFormat" >
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="browserTab" >
      <attribute name="title" >
       <string>&amp;Browser</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="btnBack" >
           <property name="toolTip" >
            <string>Back</string>
           </property>
           <property name="text" >
            <string>&lt;&lt;</string>
           </property>
           <property name="icon" >
            <iconset resource="../helpviewer.qrc" >:/images/previous.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnHome" >
           <property name="toolTip" >
            <string>Goto home page</string>
           </property>
           <property name="statusTip" >
            <string/>
           </property>
           <property name="text" >
            <string>[]</string>
           </property>
           <property name="icon" >
            <iconset resource="../helpviewer.qrc" >:/images/buildrun.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnForward" >
           <property name="toolTip" >
            <string>Forward</string>
           </property>
           <property name="text" >
            <string>>></string>
           </property>
           <property name="icon" >
            <iconset resource="../helpviewer.qrc" >:/images/next.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>51</width>
             <height>23</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btnShowPage" >
           <property name="toolTip" >
            <string>Show location bar</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnZoomUp" >
           <property name="text" >
            <string>+</string>
           </property>
           <property name="icon" >
            <iconset resource="../helpviewer.qrc" >:/images/ajouter.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnZoomDown" >
           <property name="text" >
            <string>-</string>
           </property>
           <property name="icon" >
            <iconset resource="../helpviewer.qrc" >:/images/supprimer.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnOpenInAssitant" >
           <property name="toolTip" >
            <string>Open in Assistant</string>
           </property>
           <property name="text" >
            <string>^</string>
           </property>
           <property name="icon" >
            <iconset resource="../helpviewer.qrc" >:/images/helpassistant.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="locationBar" />
       </item>
       <item>
        <widget class="QTextBrowser" name="helpBrowser" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../helpviewer.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>btnBack</sender>
   <signal>clicked()</signal>
   <receiver>helpBrowser</receiver>
   <slot>backward()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>11</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel" >
     <x>51</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnForward</sender>
   <signal>clicked()</signal>
   <receiver>helpBrowser</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>95</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>137</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHome</sender>
   <signal>clicked()</signal>
   <receiver>helpBrowser</receiver>
   <slot>home()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>64</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>79</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpBrowser</sender>
   <signal>forwardAvailable(bool)</signal>
   <receiver>btnForward</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>89</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel" >
     <x>95</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpBrowser</sender>
   <signal>backwardAvailable(bool)</signal>
   <receiver>btnBack</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>174</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel" >
     <x>14</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnZoomDown</sender>
   <signal>clicked()</signal>
   <receiver>helpBrowser</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>311</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel" >
     <x>174</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnZoomUp</sender>
   <signal>clicked()</signal>
   <receiver>helpBrowser</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>285</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel" >
     <x>174</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
